src/api/lambdas/bedrock-api-backend/assets/updateAsset.js

```diff
@@ -45,7 +45,19 @@ async function updateDependencies(client, idField, idValue, name, body) {
       }
     }
   }
-  return body.parents;
+  if (body.uses?.length > 0) {
+    for (let i = 0; i < body.uses.length; i += 1) {
+      const dependency = body.uses[i];
+      try {
+        await client.query(
+          'INSERT INTO dependencies (asset_id, dependent_asset_id) VALUES ($1, $2)',
```

Comment:
Should this insert statement also have relation_type?

Reply:
Yes, thanks for the catch! Should be good now.